guard/resources/test-command/functions/data/template.yaml

```diff
@@ -5,17 +5,27 @@
       newServer:
         Type: AWS::New::Service
         Properties:
-          Policy: |
```

Comment:
Was this template not being used elsewhere before?

Reply:
it was being used for a unit test in test_command.rs